scripts/scil_extract_ushape.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+This script can be used to extract U-Shape streamlines.
+The main idea comes from trackvis code:
+
+pt 1: 1 st end point
+pt 2: 1/3 location on the track
+pt 3: 2/3 location on the track
+pt 4: 2nd end point
+
+Compute 3 normalized vectors:
+v1: pt1 -> pt2
+v2: pt2 -> pt3
+v3: pt3 -> pt4
+
+U-factor:dot product of  v1 X v2 and v2 X v3.
+X is the cross product of two vectors.
+----------------------------------------------------------------------------
+"""
+
+import argparse
+import json
+import logging
+
+from dipy.io.streamline import save_tractogram
+import numpy as np
+
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import (add_json_args,
+                             add_overwrite_arg,
+                             add_reference_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist,
+                             check_tracts_same_format)
+from scilpy.utils.streamlines import filter_tractogram_data
+from scilpy.tractanalysis.features import detect_ushape
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
+                                description=__doc__)
+    p.add_argument('in_tractogram',
+                   help='Tractogram input file name.')
+    p.add_argument('out_tractogram',
+                   help='Output tractogram with ushape.')
+    p.add_argument('--ufactor', default=[0.5, 1], type=float, nargs=2,
```

Comment:
Could you add a second line (or in the docstring) to explain what are those two?
What if I put 0,1 or 0,0 or 1,1 or 0.5, 0.5?
Practically, what are they for?